buildrunner/steprunner/tasks/run.py

```diff
@@ -1093,14 +1095,16 @@ def _run_post_build(self, context):
         """
         self.step_runner.log.write("Running post-build processing\n")
         post_build = self.step.post_build
-        # post build always uses the image hash--can't pull
+        temp_tag = f"buildrunner-post-build-tag-{str(uuid.uuid4())}"
```

Comment:
So if I'm reading this right, this always just uses python on whales even in the legacy builder, right? That should be fine, just curious.

Reply:
Yes it will always use python on whales to do the tagging and removing. The actual build portion will be determined by the use-legacy-builder flag.